### PR TITLE
Fix #2404 : Ktlint maxline profile test package

### LINKED ISSUE #2404: Resolve ktlint max line in profile test package
**Describe the bug**
We are preventing the 100 char length at some places like test case fun name. 
`// ktlint-disable max-line-length`

**Expected behavior**
We need to remove this prevention and fix the function name under 100 char length.

### PR DESCRIPTION
**Pull request by varunvjha**
## Explanation

Fixes #2404 : Removed ' // ktlint-disable max-line-length ' and fixed the functions name under 100 char length.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by anandwana001** [comment]
@varunvjha please resolve the conflicts.

**Comment by varunvjha** [comment]
@anandwana001 PTAL.

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 1]
```diff
@@ -130,7 +130,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_inputName_clickCreate_checkOpensProfileChooserActivity() {
+  fun testAddProfileActivity_inputName_createOpensProfileChooserActivity() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: _opensProfileChooserActivity

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: ditto for all

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 2]
```diff
@@ -182,7 +182,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_changeConfiguration_clickOnCheckbox_createPin_checkIsDisplayed() {
+  fun testAddProfileActivity_configChange_clickOnCheckbox_createPin_checkIsDisplayed() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: testAddProfileActivity_configChange_clickOnCheckbox_inputPinIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 3]
```diff
@@ -210,7 +210,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_changeConfiguration_createPin_checkNotVisible() {
+  fun testAddProfileActivity_configChange_createPin_checkNotVisible() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: testAddProfileActivity_configChange_inputPinIsNotDisplayed

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: similar changes to others.

**Review by anandwana001 — CHANGES_REQUESTED** [review]
left few changes @varunvjha

**Comment by varunvjha** [comment]
@anandwana001 Could you PTAL again. The requested changes are already there.

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 4]
```diff
@@ -224,7 +224,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_inputName_inputPin_clickCreate_checkOpensProfileActivity() {
+  fun testAddProfileActivity_inputName_inputPin_opensProfileActivity() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: testAddProfileActivity_inputNameAndPin_opensProfileActivity

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt [hunk 5]
```diff
@@ -140,7 +140,7 @@ class AdminAuthActivityTest {
   }
 
   @Test
-  fun testAdminAuthActivity_inputCorrectPassword_clickImeActionButton_opensAddProfileActivity() {
+  fun testAdminAuthActivity_inputCorrectPassword_imeActionOpensAddProfileActivity() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt]: testAdminAuthActivity_inputCorrectPassword_imeAction_opensAddProfileActivity

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt [hunk 6]
```diff
@@ -189,7 +189,7 @@ class AdminAuthActivityTest {
   }
 
   @Test
-  fun testAdminAuthActivity_inputCorrectPassword_clickImeActionButton_opensAddAdministratorControlsActivity() { // ktlint-disable max-line-length
+  fun testAdminAuthActivity_inputCorrectPassword_imeActionAddAdminControlsActivity() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt]: testAdminAuthActivity_inputCorrectPassword_imeAction_opensAdminControlsActivity

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt [hunk 7]
```diff
@@ -299,7 +299,7 @@ class AdminAuthActivityTest {
   }
 
   @Test
-  fun testAdminAuthActivity_inputIncorrectPassword_inputAgain_clickImeActionButton_checkErrorIsGone() { // ktlint-disable max-line-length
+  fun testAdminAuthActivity_inputIncorrectPasswordTwice_imeActionErrorIsGone() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt]: testAdminAuthActivity_inputIncorrectPasswordTwice_imeAction_errorIsGone

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt [hunk 8]
```diff
@@ -526,7 +526,7 @@ class AdminAuthActivityTest {
 
   // TODO(#962): Reenable once IME_ACTIONS work correctly on ProfileInputView.
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt]: remove this TODO

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminPinActivityTest.kt [hunk 9]
```diff
@@ -123,7 +123,7 @@ class AdminPinActivityTest {
   }
 
   @Test
-  fun testAdminPinActivity_inputPin_inputConfirmPin_clickSubmit_checkOpensAddProfileActivity() {
+  fun testAdminPinActivity_inputPin_submitOpensAddProfileActivity() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AdminPinActivityTest.kt]: testAdminPinActivity_inputPin_submit_opensAddProfileActivity

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AdminPinActivityTest.kt]: ditto for all

**Review by anandwana001 — CHANGES_REQUESTED** [review]

**Comment by rt4914** [comment]
@varunvjha Any updates on this?

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 10]
```diff
@@ -452,7 +452,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_changeConfiguration_inputNotUniqueName_clickCreate_inputName_checkErrorIsCleared() { // ktlint-disable max-line-length
+  fun testAddProfileActivity_configChange_inputNotUniqueName_create_inputName_errorCleared() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: `_errorIsCleared`

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 11]
```diff
@@ -564,7 +564,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_changeConfiguration_inputNameWithNumbers_clickCreate_inputName_checkErrorIsCleared() { // ktlint-disable max-line-length
+  fun testAddProfileActivity_configChange_inputNameWithNumbers_create_inputName_errorCleared() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: `_errorIsCleared`

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 12]
```diff
@@ -703,7 +703,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_changeConfiguration_inputShortPin_clickCreate_inputPin_checkErrorIsCleared() { // ktlint-disable max-line-length
+  fun testAddProfileActivity_configChange_inputShortPin_create_inputPin_errorCleared() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: `_errorIsCleared`

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 13]
```diff
@@ -901,7 +901,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_changeConfiguration_inputWrongConfirmPin_inputConfirmPin_checkErrorIsCleared() { // ktlint-disable max-line-length
+  fun testAddProfileActivity_configChange_inputWrongConfirmPinAndConfirmPin_errorCleared() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: `_errorIsCleared`

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: ditto for all

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 14]
```diff
@@ -973,7 +973,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_changeConfiguration_inputPin_checkAllowDownloadNotClickable() {
+  fun testAddProfileActivity_configChange_inputPin_checkAllowDownloadNotClickable() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: testAddProfileActivity_configChange_inputPin_allowDownloadIsNotClickable

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 15]
```diff
@@ -1038,7 +1038,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_changeConfiguration_inputPin_inputConfirmPin_checkAllowDownloadClickable() { // ktlint-disable max-line-length
+  fun testAddProfileActivity_configChange_inputPin_checkAllowDownloadClickable() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: testAddProfileActivity_configChange_inputPin_allowDownloadIsClickable

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 16]
```diff
@@ -1135,7 +1135,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_inputName_changeConfiguration_checkNameIsDisplayed() {
+  fun testAddProfileActivity_inputName_configChange_checkNameDisplayed() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: testAddProfileActivity_inputName_configChange_nameIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 17]
```diff
@@ -1158,7 +1158,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_inputPin_changeConfiguration_checkPinIsDisplayed() {
+  fun testAddProfileActivity_inputPin_configChange_checkPinDisplayed() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: testAddProfileActivity_inputPin_configChange_pinIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 18]
```diff
@@ -1326,7 +1326,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_inputNotUniqueName_clickCreate_changeConfiguration_checkErrorMessageDisplayed() { // ktlint-disable max-line-length
+  fun testAddProfileActivity_inputNotUniqueName_create_configChange_errorMessageDisplayed() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: testAddProfileActivity_inputNotUniqueName_create_configChange_errorMessageIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 19]
```diff
@@ -1415,7 +1415,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_inputPin_inputDifferentConfirmPin_clickCreate_changeConfiguration_checkErrorMessageDisplayed() { // ktlint-disable max-line-length
+  fun testAddProfileActivity_inputPin_create_configChange_errorMessageDisplayed() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: testAddProfileActivity_inputPin_create_configChange_errorMessageIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt [hunk 20]
```diff
@@ -239,7 +239,7 @@ class AdminAuthActivityTest {
   }
 
   @Test
-  fun testAdminAuthActivity_inputIncorrectPassword_clickImeActionButton_checkError() {
+  fun testAdminAuthActivity_inputIncorrectPassword_imeAction_checkError() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt]: testAdminAuthActivity_inputIncorrectPassword_imeAction_errorIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt [hunk 21]
```diff
@@ -299,7 +299,7 @@ class AdminAuthActivityTest {
   }
 
   @Test
-  fun testAdminAuthActivity_inputIncorrectPassword_inputAgain_clickImeActionButton_checkErrorIsGone() { // ktlint-disable max-line-length
+  fun testAdminAuthActivity_inputIncorrectPasswordTwice_imeAction_errorIsGone() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt]: testAdminAuthActivity_inputIncorrectPassword_correct_imeAction_errorIsGone

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt [hunk 22]
```diff
@@ -524,9 +524,8 @@ class AdminAuthActivityTest {
     }
   }
 
-  // TODO(#962): Reenable once IME_ACTIONS work correctly on ProfileInputView.
   @Test
-  fun testAdminAuthActivity_inputIncorrectPasswordLandscape_clickImeActionButton_checkError() {
+  fun testAdminAuthActivity_inputIncorrectPasswordLandscape_imeAction_checkError() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt]: testAdminAuthActivity_inputIncorrectPassword_imeAction_configChange_errorIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminPinActivityTest.kt [hunk 23]
```diff
@@ -836,7 +836,7 @@ class AdminPinActivityTest {
   }
 
   @Test
-  fun testAdminPinActivity_configurationChange_inputPin_inputWrongConfirmPin_clickImeActionButton_inputConfirmPin_checkErrorIsCleared() { // ktlint-disable max-line-length
+  fun testAdminPinActivity_configChange_inputPinAndWrongConfirmPin_imeAction_inputConfirmPin() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AdminPinActivityTest.kt]: testAdminPinActivity_configChange_inputWrongConfirmPin_imeAction_correct_errorIsCleared

File: app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt [hunk 24]
```diff
@@ -126,7 +126,7 @@ class PinPasswordActivityTest {
   }
 
   @Test
-  fun testPinPasswordActivityWithAdmin_checkKeyboardIsVisibleByDefault() {
+  fun testPinPasswordActivityWithAdmin_keyboardIsVisibleByDefault() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt]: testPinPasswordActivity_withAdmin_keyboardIsVisibleByDefault

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt]: ditto for all

File: app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt [hunk 25]
```diff
@@ -447,7 +447,7 @@ class PinPasswordActivityTest {
   }
 
   @Test
-  fun testPinPasswordActivityWithUser_clickForgot_inputAdminPin_clickSubmit_changeConfiguration_restPinDialogIsDisplayed() { // ktlint-disable max-line-length
+  fun testPinPasswordActivityWithUser_forgot_inputAdminPin_submit_configChange_resetPinDisplayed() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt]: testPinPassword_withUser_forgot_inputAdminPin_submit_configChange_resetPinIsDisplayed

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt]: we can use `testPinPassword_` as a prefix in all test name for this file.

File: app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt [hunk 26]
```diff
@@ -643,7 +643,7 @@ class PinPasswordActivityTest {
   }
 
   @Test
-  fun testPinPasswordActivityWithAdmin_inputWrongPin_changeConfiguration_checkIncorrectPinShows() {
+  fun testPinPasswordActivityWithAdmin_inputWrongPin_configChange_incorrectPinShows() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt]: testPinPasswordActivity_withAdmin_inputWrongPin_configChange_incorrectPinIsDisplayed

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Thanks @varunvjha

nit changes suggested.

Also, it will be helpful if you comment done and something in all the suggested changes to get a better understanding if the suggested changes are fulfilled or need any help.

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt (hunk 10 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt (hunk 11 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt (hunk 12 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt (hunk 13 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt (hunk 14 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt (hunk 15 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt (hunk 16 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt (hunk 17 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt (hunk 18 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt (hunk 19 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt (hunk 20 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt (hunk 21 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt (hunk 22 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt (hunk 24 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt]: Done for all

File: app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt (hunk 25 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt]: Done for all

File: app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt (hunk 26 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt]: Done

**Comment by Luffy18346** [comment]
@varunvjha please keep assignee section up-to-date. Take a look at 4th point in oppia instructions [https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up](url).

Remove yourself from assigneed and assign it to appropriate person.

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt [hunk 27]
```diff
@@ -206,7 +206,7 @@ class AdminAuthActivityTest {
         )
       ).perform(
         editTextInputAction.appendText("12345"),
-        pressImeActionButton()
+        pressimeAction()
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt]: why this is modified?

**Review by anandwana001 — CHANGES_REQUESTED** [review]

File: app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt (hunk 27 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt]: It was a mistake on my part which is corrected now.

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @varunvjha

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**